docs/tutorial/notifications.md

```diff
@@ -101,41 +111,41 @@ handling replies requires the use of
 which helps with registering the required COM components and calling your
 Electron app with the entered user data.
 
-#### Quiet Hours / Presentation Mode
```

Comment:
Apparently Quiet Hours got renamed to "Focus Assist" in Windows 11. I'm just going to simplify this section name by calling it "notification state".